android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -266,7 +282,7 @@ public void onUrlDeviceDiscoveryUpdate() {
       @Override
       public void run() {
         for (PwPair pwPair : mPwCollection.getGroupedPwPairsSortedByRank(
```

Comment:
Is this still called `SortedByRank`?  Please make a change to remove `Rank` from these method names.